admin/c2cgeoportal_admin/templates/404.jinja2

```diff
@@ -0,0 +1,8 @@
+{% extends "layout.jinja2" %}
```

Comment:
The admin interface use jinnja?

Reply:
We decided it would depend on jinja2, yes.

Reply:
Yes, it's an old comment :-)